### PR TITLE
Feat gear2

### PR DESCRIPTION
**Pull request by mahoyen**

### CODE REVIEW

File: controller.c [hunk 1]
```diff
@@ -13,6 +13,9 @@
 #include "pid.h"
 #include "controller.h"
 
+#define MAX_DUTY_BOUND 95
+#define MIN_DUTY_BOUND 5
```
Comment (haavardfiskaa) [inline controller.c]:
```suggestion
#define MIN_DUTY_BOUND 50
```

File: controller.c [hunk 2]
```diff
@@ -31,64 +34,70 @@ void set_I(uint8_t duty)
...
-	if (f32_DutyCycleCmd < 50)// bounding at 50 to prevent rheostatic braking and backwards motion
+	if (f32_DutyCycle < MIN_DUTY_BOUND)// bounding at 50 to prevent rheostatic braking and backwards motion
 	{
-		f32_DutyCycleCmd = 50;
+		f32_DutyCycle = MIN_DUTY_BOUND;
 	}
 	
+	if (vals->gear_status == GEAR2)
+	{
+		f32_DutyCycleCmd = (1-f32_DutyCycle);
```
Comment (haavardfiskaa) [inline controller.c]:
```suggestion
		f32_DutyCycleCmd = (100-f32_DutyCycle);
```

**Review by haavardfiskaa — COMMENTED** [review]
Looks good, should be tested without motor the first times.

**Comment by mahoyen** [comment]
Will do testing, waiting with merging.

**Pull request closed without merging**